File: CHAP -- @NUMERIC PROBLEM/CHAP -- BlueJCodes/SourceCode-JCreatorPro/PracticeCode/src/Encapsulation-DataHiding/PrivateDataMember.java
import java.io.*;

class Encapsulation
{
	private int x,y,z;

	InputStreamReader read = new InputStreamReader(System.in);
	BufferedReader in = new BufferedReader(read);

	void getValue()throws IOException
	{
		System.out.print("Enter the vlaue of x: ");
		x = Integer.parseInt(in.readLine());

		System.out.print("Enter the vlaue of y: ");
		y = Integer.parseInt(in.readLine());
	}

	void addition()
	{
		z = (x + y);
	}

	void display()
	{
		System.out.println("Sum is: " + z);
	}
}

class PrivateDataMember
{
	public static void main (String[] args)throws IOException
	{
		Encapsulation obj = new Encapsulation();

		obj.getValue();
		obj.addition();
		obj.display();
	}
}

File: CHAP -- @Notes/java/RecapsulationCodes/RecapsulationCodes/Encapsulation-DataHiding/PrivateDataMember.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RecapsulationCodes.Encapsulation_DataHiding
{
    class Encapsulation
    {
        private int x, y, z;

        public void getValue()
	    {
		    Console.Write("Enter the vlaue of x: ");
		    x = int.Parse(Console.ReadLine());

            Console.Write("Enter the vlaue of y: ");
            y = int.Parse(Console.ReadLine());
	    }

	    public void addition()
	    {
		    z = (x + y);
	    }

	    public void display()
	    {
            Console.WriteLine();
            Console.WriteLine("Sum is: " + z);
	    }
    }
    
    class PrivateDataMember
    {
        public static void Main(string[] args)
        {
            Encapsulation obj = new Encapsulation();

            obj.getValue();
            obj.addition();
            obj.display();

            Console.WriteLine();
            Console.WriteLine("Please any key to continue...!");
            Console.ReadKey(true);
        }
    }
}
